pkg/cassandra/nodetool/client/client.go

```diff
@@ -0,0 +1,89 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/pborman/uuid"
+)
+
+const (
+	StorageServicePath = "read/org.apache.cassandra.db:type=StorageService"
+)
+
+type StorageService struct {
+	HostIdMap        map[string]uuid.UUID
+	LiveNodes        []string
+	UnreachableNodes []string
+	LeavingNodes     []string
+	JoiningNodes     []string
+	MovingNodes      []string
+	LocalHostId      uuid.UUID
+}
```

Comment:
From what I can tell, this struct is deseralised into from JSON? If so, please add `json:""` struct tags so it's explicit. I guess `uuid.UUID` defines its own Marshal/Unmarshal methods?